src/message/basicMessage.js

```diff
@@ -127,7 +128,7 @@ class BasicMessage{
 
     /**
      * Sets the Api Template for the message.
-     * (Optional) Either textBody or HtmlBody must be used or use a ApiTemplate
+     * (Optional) Either textBody or HtmlBody must be used or use a ApiTemplate with an AmpBody
```

Comment:
rephrase this line: "Either TextBody or HtmlBody must be used with the AmpBody or use a ApiTemplate"